### ISSUE TITLE
Why emulate LM?

### ISSUE THREAD

**Issue by Stage4001** [spencermwoo/lordsmobile#1]
This method could be a lot faster if you werent running lords mobile inside bluestacks. How about you try it using the Steam version of lords mobile instead?

**Comment by Stage4001** [comment]
Another thing, the binary for LM is practically freely available of you look hard enough, a much faster method could be to simply write a code that sends commands to IGG pretending to be a user scanning the map, it would then send you all the data in an already easy to parse format at insane speed.

**Comment by MantisGalaxy** [comment]
True

**Comment by Stage4001** [comment]
If this is something someone wants to do (sending and receiving packets to/from the IGG server to make a map scanner let me know and I can give you the info you need to make it happen.

**Comment by MantisGalaxy** [comment]
If this does happen, I will like to have the end product of this, that project will be really helpful

**Comment by Stage4001** [comment]
Same I dont have the skills to do it myself but know where to find the relevant info.

**Comment by Stage4001** [comment]
Their encryption key for their data packets is JeffHappy, has been for years.

**Comment by MantisGalaxy** [comment]
Oh, wow. That's interesting. Do you play LM ?

**Comment by MantisGalaxy** [comment]
Why haven't they changed their encryption keys, wouldn't that be a massive data leak

**Comment by Stage4001** [comment]
x AntiHero x

**Comment by Stage4001** [comment]
No, all it does is encrypt the data between you and the lm server so its not like it matters if you read that and it would be more work for them to try to keep changing it than its worth, the only thing you can do with that encryption key is make a bot that plays the game.

**Comment by Stage4001** [comment]
If you want to discuss this further you can add me on Discord. Stage4000#4099

**Comment by MantisGalaxy** [comment]
Added!
